comms/core/src/connectivity/test.rs

```diff
@@ -423,7 +423,7 @@ async fn pool_management() {
         if conn != important_connection {
             assert_eq!(conn.handle_count(), 2);
             // The peer connection mock does not "automatically" publish event to connectivity manager
-            conn.disconnect(Minimized::No).await.unwrap();
+            let _unused = conn.disconnect(Minimized::No).await;
```

Comment:
if the disconnect fails, the test will now pass, where as before it would fail. 
Is there a reason for this change?

Reply:
Hmmm... it seems I chose the easy way out. It turns out that the peer_connection mock's _disconnect_ behaved differently than the actual _disconnect_, which became evident due to the finer grain of error handling. The mock propagated any errors other than the `YamuxControlError::ConnectionClosed` error, whereas the actual implementation silences all errors.

This will be fixed in the new commit.